src/main/kotlin/io/github/karmasmp/karma/chat/ChatUtils.kt

```diff
@@ -96,31 +99,22 @@ object ChatUtils {
 object GlobalRenderer : ChatRenderer {
     override fun render(source: Player, sourceDisplayName: Component, message: Component, viewer: Audience): Component {
         val playerHead = Noxesium.buildSkullComponent(source.uniqueId, false, 0, 0, 1.0f)
+        val plainMessage = PlainTextComponentSerializer.plainText().serialize(message)
         if(source.hasPermission("karma.group.admin")) {
-            return if(source.isInStaffMode()) {
-                playerHead
-                    .append(Component.text(Formatting.Prefix.ADMIN_PREFIX.value))
-                    .append(sourceDisplayName.color(NamedTextColor.DARK_RED))
-                    .append(Component.text(" SM", NamedTextColor.GOLD))
-                    .append(Component.text(": "))
-                    .append(message)
-            } else {
-                playerHead
-                    .append(Component.text(Formatting.Prefix.ADMIN_PREFIX.value))
+            return playerHead
+                    .append(Component.text(Formatting.Prefix.ADMIN_PREFIX.value)).append(Component.space())
                     .append(sourceDisplayName.color(NamedTextColor.DARK_RED))
                     .append(Component.text(": "))
-                    .append(message)
-            }
+                    .append(allTags.deserialize(plainMessage))
         } else {
             val lifeCount = source.getKarmaLives()
-
-            val lives = ChatUtils.livesAsComponent(lifeCount).append(Component.text(" "))
+            val lives = ChatUtils.livesAsComponent(lifeCount).append(Component.space())
 
             return playerHead
                 .append(lives)
                 .append(sourceDisplayName)
-                .append(Component.text(": "))
-                .append(message)
+                .append(if(source.isCreator()) { restrictedTags.deserialize(" <red>[LIVE]<reset>: ") } else { restrictedTags.deserialize(": ") })
```

Comment:
I would maybe call it isLive instead of isCreator, since everyone is a creator and what I originally thought of was more of a "who is currently live/recording" indicator

Reply:
Renamed.